examples/classics/tetris/.gitignore

```diff
@@ -0,0 +1,2 @@
+.zig-cache/
+zig-out/
```

Comment:
Let's make these global in the top-level .gitignore instead of adding an extra file here